bitwarden_license/src/app/policies/maximum-vault-timeout.component.html

```diff
@@ -0,0 +1,27 @@
+<app-callout type="tip" title="{{'prerequisite' | i18n}}">
+    {{'requireSsoPolicyReq' | i18n}}
```

Comment:
These strings seem poorly named now. It's not really SSO, it's just single org

Reply:
If we rename it, all locales needs to be updated which feels unnecessary.

Reply:
You're right, but we need to be able to have these evolve over time.

I made a quick PR to solve this problem here: https://github.com/bitwarden/jslib/pull/484

The usage would be:

`messages.json:`
```
{
  ...
  "requireSingleOrgPolicy" : {
    "message": "..."
    ...
    "previousNames": [
      "requireSsoPolicyReq"
    ]
  }
  ...
}
```